flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -234,6 +234,13 @@
 			.withDeprecatedKeys("taskmanager.net.client.connectTimeoutSec")
 			.withDescription("The Netty client connection timeout.");
 
+	@Documentation.Section(Documentation.Sections.ALL_TASK_MANAGER_NETWORK)
+	public static final ConfigOption<Integer> CLIENT_CONNECT_RETRY =
+		key("taskmanager.network.netty.clientRetry")
+			.defaultValue(0)
+			.withDeprecatedKeys("taskmanager.network.netty.clientRetry")
+			.withDescription("The retry times for building netty client connection.");
+
```

Comment:
I think this option might be more general than just netty and not only for "client connect retries". Also I think `taskmanager.network.netty` are the properties that are passed directly to netty config, am I right? 

So I would rename it to `taskmanager.network.retries`/`NETWORK_RETRIES`/`The number of retry attempts for network communication. Currently it's only used for establishing input/output channel connections.`

Reply:
Yeah, it is a better  config name